src/game_modes.c

```diff
@@ -74,13 +74,13 @@ StateFunc gameModeMap[] = {
 };
 
 // BSS
-extern s16 D_800A08F0;
+extern s16 gGameMode;
```

Comment:
Can we call this gameMode instead of gGameMode? The BSS location & the set and get functions imply that its `static` and local - not global - to this file